soccer_vision_msgs/msg/MarkingSegment.msg

```diff
@@ -0,0 +1,15 @@
+# Describes a detected straight segment marking from vision modules, in image coordinates.
+# Note that curved segments can be approximated with a number of small segments, using
+# this msg.
+
+float32 CONFIDENCE_UNKNOWN=-1
+
+# Two points defining the vector of the segment. The center is orthogonally in the middle of the segment
+# The points should be located at the centre along the width of the segment marking.
```

Comment:
I am a little bit confused about this one. What center is where?

Reply:
@jaagut I've updated the comment. Does that make more sense?